src/directory/directory.mjs

```diff
@@ -1467,6 +1467,11 @@ export const directory = {
             route: '/lib-v1/storage/configureaccess',
             filters: ['android', 'ios', 'flutter', 'js']
           },
+          {
+            title: 'Use Transfer Acceleration',
+            route: '/lib-v1/storage/transfer-acceleration',
+            filters: ['android', 'js', 'react-native', 'flutter']
```

Comment:
If this is an entirely new page for the `lib-v1` route, shouldn't we only add it to JS?

Reply:
We have this page in [V5 as well](https://github.com/aws-amplify/docs/pull/5906/files/66f04742df2707d711a641ff2d8718d006ec173b#diff-782c64a4a6ba1f0bff31bf639532b0169bd826b3e4208dc69ed6af06a98b08aeL797). I've just updated the route

Reply:
ahh makes sense, updated